source/classes.tex

```diff
@@ -4201,6 +4179,27 @@
 \tcode{circle::draw()} must be provided.
 \end{example}
 
+\pnum
+\begin{note}
+In general, an abstract class type can only be used
+\begin{itemize}
+\item
+as the referred to type in a reference,
+\item
+as the pointed to type in a pointer,
+\item
+in an explicit type conversion
+to the above types~(\ref{expr.static.cast}, \ref{expr.reinterpret.cast},
+\ref{expr.const.cast}, \ref{expr.dynamic.cast}), or
```

Comment:
That's an odd point to make, given that you already highlighted that you can form pointers and references to abstract class types.